docs/source/notebooks/03_Extract_geographical_watershed_properties.ipynb

```diff
@@ -9,7 +9,9 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    grid=rio.open_rasterio(rasterio.MemoryFile(grid0[0]).open())
</pre><p>nitpick but use spaces around "=" for normal assigments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>